InWorldz/InWorldz.Phlox.Engine/LSLSystemAPI.cs

```diff
@@ -2662,7 +2662,7 @@ public string iwFormatTime(int unixtime, int isUTC, string format)
                 date = DateTime.SpecifyKind(date, DateTimeKind.Local);
             }
 
-            if (format == "")
+            if (String.IsNullOrEmpty(format))
```

Comment:
I'm reasonably certain that LSL can't send a null string, and even if it did the result would be that you'd get a properly formatted date instead of an exception.